exercises/binary_converter.cpp

```diff
@@ -5,3 +5,29 @@
   Insert first number: 8
   The binary number is: 1000
 */
+
+#include <iostream>
+using namespace std;
+
+int main() {
+    int a[10], n, i;
+
+    // prendo il numero da convertire in binario
+    cout << "Insert first number: ";
+    cin >> n;
+
+    // mi ricavo il binario del numero
+    for (i = 0; n > 0; i++) {
+        a[i] = n % 2;
+        n = n / 2;
+    }
+
+    // stampo il numero
+    cout << "The binary number is: ";
+    for (i = i - 1; i >= 0; i--) {
+        cout << a[i];
+    }
+
+    cout << endl;
+    return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```